docs/source/concepts/Materials.rst

```diff
@@ -0,0 +1,104 @@
+.. _Materials:
+
+Materials
+=========
+
+.. contents::
+
+Neutron scattering lengths and cross sections of the elements and their
+isotopes have been taken from
+`NIST <http://www.ncnr.nist.gov/resources/n-lengths/list.html>`__.
+
+Chemical Composition with Examples
+##################################
+- ``H2 O`` - Isotopically averaged Hydrogen
+- ``(H2)2 O`` - Heavy water
+- ``D2 O`` - Another way to specify heavy water
+
+Enter a composition as a molecular formula of elements or isotopes.
+For example, basic elements might be ``H``, ``Fe`` or ``Si``, etc.
+A molecular formula of elements might be ``H4-N2-C3``, which
+corresponds to a molecule with 4 Hydrogen atoms, 2 Nitrogen atoms and
+3 Carbon atoms.  Each element in a molecular formula is followed by
+the number of the atoms for that element, specified **without a hyphen**,
+because each element is separated from other elements using a hyphen.
+
+The number of atoms can be integer or float, but must start with a
+digit, e.g. 0.6 is fine but .6 is not. This can be used to set elemental ratios
+within a chemical composition. For example 95.1% Vanadium 4.9% Niobium can be
+expressed as ``V0.951 Nb0.049``. *Warning: Using this representation will
+calculate all properties except for SampleNumberDensity which must be
+set manually if required*
+
+Isotopes may also be included in a :py:obj:`material
+<mantid.kernel.Material>` composition, and can be specified alone (as
+in ``Li7``), or in a molecular formula (as in ``(Li7)2-C-H4-N-Cl6``).
```

Comment:
I think the single lithium example should read ``(Li7)`` instead of ``Li7``, see issue #21467. The former is treated as an isotope specification while the latter means a molecule of 7 lithium atoms. Fixing this would crush the above issue as well.